CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
+## 0.3.4
+
+* Correctly dispose the ios health kit plugin when the app is in the background
+
 ## 0.3.0
 
 * **Breaking** - `UserService.getAll` return type change from
-`List<User>` to `GetAllUsersResponse`
+  `List<User>` to `GetAllUsersResponse`
```

Comment:
Extra space?

Reply:
I have formatted the file and this was added by the formatter. There is no change for the person who is looking at the change log.